content/ember-data/v3/najax.md

```diff
@@ -0,0 +1,49 @@
+---
+id: ember-data:najax-fallback
+title: Deprecate najax request
+until: '4.0.0'
+since: '3.22.0'
+---
+#### Deprecates the adapter `najax`
```

Comment:
I wonder if this could be phrased slightly differently, this sounds to me like an adapter class being deprecated